pyproject.toml

```
@@ -62,6 +62,7 @@ package = "./pypdf"
 
 [tool.flit.sdist]
 exclude = [".github/*", "docs/*", "sample-files/.github/*", "sample-files/.gitignore", "sample-files/.pre-commit-config.yaml",  "requirements/*", ".flake8", ".gitignore", ".gitmodules", ".pylintrc", "tox.ini", "make_release.py", "mutmut-test.sh", ".pre-commit-config.yaml", ".gitblame-ignore-revs", "Makefile", "mutmut_config.py"]
+include = ["resources/", "tests/", "sample-files/"]
```

I recommend to not include the sample files here - they can be downloaded regularly as an archive and do not change very often. Otherwise, this adds additional bloat and would require adapting the PyPI workflow.

```suggestion
include = ["resources/", "tests/"]
```